1-js/02-first-steps/12-while-for/2-which-value-while/task.md

```diff
@@ -2,19 +2,19 @@ importance: 4
 
 ---
 
-# Which values does the while loop show?
+# Quais valores o loop while retorna?
 
-For every loop iteration, write down which value it outputs and then compare it with the solution.
+Para cada iteração dos loops, anote o valor que ele retorna, depois compare com a solução.
 
-Both loops `alert` the same values, or not?
```

Comment:
I think would be a good a idea to ask to add the word return here. As you translated.